app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -437,7 +452,17 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
             binding.player.setCurrentChapterName()
         }
 
-        playVideo()
+
+        localSearchResult?.let {
+            PlayOfflineDialog(it, {
+                (requireActivity() as MainActivity).supportFragmentManager.commit {
+                    remove(this@PlayerFragment)
+                }
+            }) { _, _ ->
+                playVideo()
+            }.show(parentFragmentManager, null)
+
+        } ?: playVideo()
```

Comment:
```suggestion
        if (localSearchResult != null) {
            PlayOfflineDialog(it, {
                (requireActivity() as MainActivity).supportFragmentManager.commit {
                    remove(this@PlayerFragment)
                }
            }) { _, _ ->
                playVideo()
            }.show(parentFragmentManager, null)

        } else {
            playVideo()
        }
```
I'd prefer using normal if conditions for readability